FoodBookApp/UI/Views/CreateReview/CreateReview1View.swift

```diff
@@ -17,85 +17,90 @@ struct CreateReview1View: View {
     let customGray2 = Color(red: 242/255, green: 242/255, blue: 247/255)
     
     var body: some View {
-        VStack {
-            VStack(alignment: .leading){
-                
-                // Header
-                HStack{
-                    TextButton(text: "Cancel", txtSize: 20, hPadding: 0) // FIXME: should redirect
-                    Spacer()
-                    Text("Review")
-                        .bold()
-                        .font(.system(size: 20))
-                    Spacer()
-                    BoldTextButton(text: "Next", txtSize: 20) { print("Next") } // FIXME: should redirect and verify that at least one has been selected
+        NavigationView {
+            VStack {
+                VStack(alignment: .leading){
+                    // Header
+                    HStack{
+                        TextButton(text: "Cancel", txtSize: 17, hPadding: 0) // FIXME: should redirect
```

Comment:
This button also has no navigation (see video)

https://github.com/ISIS3510-202410-Team23/SwiftApp/assets/69609680/9ef8b60e-5fe4-4f9f-bded-b743a1ffe1a3